src/pages/whats-new.js

```diff
@@ -0,0 +1,131 @@
+    const {title, node_locale} = data.contentfulPage
+    const changelog = data.allContentfulChangelogContent.nodes;
+    return (
+    <Layout pageClassName="whats-new-page">
+        <SEO title={title} />
+        <Container>
+        <Row><Col><h1>{data.contentfulPage.title}</h1></Col></Row>
+        <Row className="d-flex bd-highlight">
+                <Col md={4} className="d-none d-sm-block p-2 w-100 bd-highlight" id="left-dates">
+                    <ListGroup variant="flush" className="mt-3">
+                    {
+                        changelog.map((feature,i) => (
+                            <Link
+                            key={i}
+                            className="btn"
+                            activeClass="active"
+                            spy={true}
+                            smooth={true}
+                            offset={-150}
+                            duration= {550} 
+                            to={dateToDivID(dateToString(feature.date,node_locale))}>
+                                <ListGroup.Item action >
+                                    {dateToString(feature.date,node_locale)}
+                                </ListGroup.Item>
+                            </Link>
+                        ))
+                    }
+                    </ListGroup>
+                </Col>
+                <Col md={8} className="px-3 flex-shrink-1 bd-highlight" id="right-content">
+                    {
+                        changelog.map(feature => (
+                            <Element name={dateToDivID(dateToString(feature.date,node_locale))}  style={{marginBottom: '80px'}}>
+                                <React.Fragment>
+                                <NewFeature
+                                    heading={feature.heading}
+                                    date={dateToString(feature.date,node_locale)}
+                                    description={documentToReactComponents(feature.description.json, options)}
+                                    pagePath={feature.pagePath}
+                                    image={feature.imageChangelog ? 'https:' + getNestedObject(feature.imageChangelog, ['file', 'url']): ''}
+                                    image_title={feature.imageChangelog ? getNestedObject(feature.imageChangelog, ['file', 'title']): ''}
+                                />
+                                </React.Fragment>
+                            </Element>
+                        ))}
+                </Col>
+            </Row>
+        </Container>
+    </Layout>
+)}
+
+export default WhatsNewPage
+
+export const query = graphql`
+query WhatsNewPage($language: String) {
+    site(siteMetadata: {}) {
+      siteMetadata {
+        latestSchoolYear
+      }
+    }
+    contentfulPage(slug: {eq: "whats-new"}, node_locale: {eq: $language}) {
+      slug
+      title
+      node_locale
+    }
+    allContentfulChangelogContent(filter: {node_locale: {eq: $language}}) {
```

Comment:
it would be cool to display the changes from newest to oldest. If we change the sort order here, will the order also change in the display?

Reply:
I now have it sorted from the graphql call 🎉, what display ?
![image](https://user-images.githubusercontent.com/16183561/92563528-9164b300-f22c-11ea-9437-543674427753.png)

Reply:
oh sorry, I thought when I was looking at it yesterday it was in the reverse order. cool!